Xamarin.Forms.Core/PlatformConfiguration/WindowsSpecific/RefreshView.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace Xamarin.Forms.PlatformConfiguration.WindowsSpecific
+{
+	using FormsElement = Forms.RefreshView;
+
+	public static class RefreshView
+	{
+		public enum RefreshPullDirection
+		{
+			LeftToRight,
+			TopToBottom,
+			RightToLeft,
+			BottomToTop
+		}
+
+		public static readonly BindableProperty RefreshPullDirectionProperty = BindableProperty.Create("RefreshPullDirection", typeof(RefreshPullDirection), typeof(FormsElement), RefreshPullDirection.TopToBottom);
```

Comment:
I see only the enum was added to the "New APIs" in the description. Do we need to include the property itself as well?

Reply:
Yes, I added it to the "New APIs" description.